src/fullscreen/fullscreen-controller.js

```diff
@@ -255,6 +255,14 @@ class FullscreenController {
       this._player.dispatchEvent(new FakeEvent(this._player.Event.EXIT_FULLSCREEN));
     }
   }
+  /**
+   * Destroys the FullScreenController.
+   * @returns {void}
+   * @public
+   */
+  destroy(): void {
+    this._player = null;
```

Comment:
`this._isInBrowserFullscreen = false;`

Reply:
also need to clean the event manager here

Reply:
I'm not sure it needs to be in this scope of change, I just detach the player refs all over the code.